exovetter/vetters.py

```diff
@@ -125,7 +126,9 @@ def plot(self):  # pragma: no cover
 
         # target is populated in TCE, assume it already exists.
         target = self.tce.get('target_name', 'Target')
-        lpp.plot_lpp_diagnostic(self.plot_data, target, self.norm_lpp)
+        fig = lpp.plot_lpp_diagnostic(self.plot_data, target, self.norm_lpp)
+
+        return fig
```

Comment:
If `plot` is supposed to return a `fig` here, it should be done to the base class and all the other child classes as well, for consistency.